web/src/features/Auth/components/SecureAdminConsole.tsx

```diff
@@ -196,7 +196,7 @@ class SecureAdminConsole extends React.Component<Props, State> {
       this.loginText.current.innerHTML,
       this.loginText.current.clientWidth,
       "32px",
-      null
+      "10px"
```

Comment:
there's a resize function in utils that will resize the app name, but this doesn't occur every time. Need to look into it more.

Reply:
interesting